analyses/data-pre-release-qc/01-data-harmonization-qc.Rmd

```diff
@@ -446,7 +446,7 @@ print(paste("DNA biospecimen in cnv-controlfreec.tsv.gz missing in histolgies =
 - GATK seg file (cnv-gatk.seg.gz)
 ```{r}
 # check that all biospecimen IDs are in the file
-biospecimen_ids <- readr::read_tsv(file.path(data_dir, "cnv-gatk.seg.gz")) %>%
+biospecimen_ids <- readr::read_tsv(file.path(data_dir, "v13", "cnv-gatk.seg.gz")) %>%
```

Comment:
For this, can you stack on top of the data release PR and use the data folder so this is not tied to a version?